src/popup.html

```diff
@@ -20,14 +20,23 @@
             font-family: sans-serif;
             font-size: 75%;
         }
+
+        a:link,
+        a:visited,
+        a:active,
+        a:hover {
+            text-decoration: none;
+        }
     </style>
 </head>
 
 <body style="width: 250px;">
-    <div style="margin-bottom: 10px;">
-        <img class="center" src="../img/icon32.png">
-        <h3 class="title">BetterMaplewood</h3>
-    </div>
+    <a href="https://hosting.maplewood.com/AB/Private/WA/WA/Maplewood/connectEd/viewer/login/login.aspx?logintype=S" target="_blank">
```

Comment:
BetterMaplewood isn't specific to a single school, so this link should be removed.
```suggestion
```